java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/SymbolicValue.java

```diff
@@ -82,6 +82,28 @@ public String toString() {
     FALSE_LITERAL
   );
 
+  private static int idGenerator;
+  private int id;
+
+  public SymbolicValue() {
+    id = idGenerator++;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS881)